gqueries/input_elements/label/relative_price_of_co2.gql

```diff
@@ -0,0 +1,8 @@
+# Relative price of CO2
+
+- query =
+    100.0 * DIVIDE(
+      SUM(QUERY_FUTURE(-> {Q(price_of_co2)}), PRODUCT(-1,QUERY_PRESENT(-> {Q(price_of_co2)}))),
```

Comment:
`PRODUCT(-1, x)` can also be written as `-x` in GQL, the same for adding `+`, dividing `/` and subtracting `-`. At least - in the Sandbox of ETEngine I'm allowed to write: `(V(households_final_demand_electricity, demand) * -1 / 5 + 2 - 10)`. So I think this can become:

```ruby
100.0 * 
((QUERY_FUTURE(-> {Q(price_of_co2)}) - QUERY_PRESENT(-> {Q(price_of_co2)})) /
QUERY_PRESENT(-> {Q(price_of_co2)})
```